python/openai/openai_frontend/engine/triton_engine.py

```diff
@@ -250,6 +250,9 @@ def _get_model_metadata(self) -> Dict[str, TritonModelMetadata]:
         for name, _ in self.server.models().keys():
             model = self.server.model(name)
             backend = model.config()["backend"]
+            if not backend:
+                # Check platform field as a backup, this will support 'ensemble' models
+                backend = model.config()["platform"]
```

Comment:
@rmccorm4  should we check against a list of  supported platform types here to rule out any accidents

Reply:
Sure, we can be strict here just in case. 

There's currently no use case I'm aware of where the platform would be used to specify a backend (ex: `platform: tensorrt_plan`) that doesn't have a backend equivalent, and we generally encourage use of backend for this purpose for pretty much everything other than ensembles - so I think it's fine to be strict for now.

Reply:
@blongnv could you modify it to be something like this?
```python
# Explicitly handle ensembles to avoid any runtime validation errors
if not backend and model.config()["platform"] == "ensemble":
  backend = "ensemble"
```

Reply:
Implemented, please check